README.md

```diff
@@ -62,20 +64,25 @@ simulator scenario list
 </pre>
 This will list all currently available scenrios
 
+_To get a better idea of what is involved in each scenario, e.g. node-shock-tactics_
+<pre>
+simulator scenario describe node-shock-tactics
+</pre>
+
 _Launch a scenario (sets up your cluster)_
 <pre>
 simulator scenario launch node-shock-tactics
 </pre>
 This will launch your selected scenario.
 
-_Login the environment_
+_Login to the environment_
 <pre>
 simulator ssh attack
 </pre>
 
-Once you run <code>simulator ssh attack</code> you will be logged into a container running on the Bastion host.  Upon login an outline of the challenge will be displayed.  In addition, short cuts for logging into the master, or nodes, of the Kubernetes cluster are displayed.  Use these shortcuts to log into the correct starting point as outlined in the challenge.  **Note** that some starting points will require you are on the cluster first to access to starting point.
+Running <code>simulator ssh attack</code> logs you into a container running on the Bastion host.  Upon login an outline of the challenge will be displayed.  In addition, short cuts for logging into the master, or nodes, of the Kubernetes cluster, how to show hints and start tasks, etc are displayed.  <code>starting_point</code> logs you into the correct starting point as per the challenge.

```

Comment:
> In addition, short cuts for logging into the master, or nodes, of the Kubernetes cluster, how to show hints and start tasks, etc are displayed

This sentence is a bit clunky, might be worth being 2 sentences? Also I think shortcuts is one word.